SwiftMailer/MessageFormat/MessagePayloadV3.php

```diff
@@ -41,7 +42,15 @@ public function getMailjetMessage(Swift_Mime_SimpleMessage $message) {
 
         // Handle attachments
         foreach ($message->getChildren() as $child) {
-            if ($child instanceof Swift_Attachment) {
+            if (($child instanceof Swift_Attachment || $child instanceof Swift_Image)
+                && $child->getDisposition() === "inline"
+            ) {
+                $inline_attachments[] = array(
+                    'Content-type' => $child->getContentType(),
+                    'Filename' => $child->getFilename(),
+                    'content' => base64_encode($child->getBody())
+                );
+            } elseif ($child instanceof Swift_Attachment) {
                 //Handle regular attachments
                 if ($child->getDisposition() === "attachment") {
```

Comment:
You can combine the condition `elseif ($child instanceof Swift_Attachment) && $child->getDisposition() === "attachment"`